pvlib/atmosphere.py

```diff
@@ -326,24 +328,24 @@ def first_solar_spectral_correction(pw, airmass_absolute,
     Spectral mismatch modifier based on precipitable water and absolute
     (pressure-adjusted) airmass.
 
-    Estimates a spectral mismatch modifier M representing the effect on
+    Estimates a spectral mismatch modifier :math:`M` representing the effect on
     module short circuit current of variation in the spectral
-    irradiance. M is estimated from absolute (pressure currected) air
+    irradiance. :math:`M`  is estimated from absolute (pressure currected) air
     mass, :math:`AM_a`, and precipitable water, :math:`Pw`, using the following
     function:
 
     .. math::
 
-        M = c_1 + c_2 AM_a  + c_3 Pw  + c_4 AMa^{0.5}
-            + c_5 Pw^{0.5} + c_6 \frac{AMa} {Pw^{0.5}}
+        M = c_1 + c_2 AM_a  + c_3 Pw  + c_4 AM_a^{0.5}
+            + c_5 Pw^{0.5} + c_6 \frac{AM_a} {Pw^{0.5}}
 
     Default coefficients are determined for several cell types with
     known quantum efficiency curves, by using the Simple Model of the
     Atmospheric Radiative Transfer of Sunshine (SMARTS) [1]_. Using
     SMARTS, spectrums are simulated with all combinations of AMa and
     Pw where:
 
-       * :math:`0.5 cm <= Pw <= 5 cm`
+       * :math:`0.5 \roman{cm} <= Pw <= 5 \roman{cm}`
```

Comment:
* **Maybe use just `\text` or `\textrm` instead?** There are so many options here unfortunately, I've seen `\textrm`, `\textnormal`, `\mathrm`, etc. and there are several [Tex SO](https://tex.stackexchange.com/) posts ([`\text` v `\mathrm`](https://tex.stackexchange.com/questions/19502/is-there-a-preference-of-when-to-use-text-and-mathrm), [`\textrm` v `\mathrm`](https://tex.stackexchange.com/questions/22350/difference-between-textrm-and-mathrm)) that have suggestions.
* I believe [`\roman`](http://www.emerson.emory.edu/services/latex/latex_10.html) and `\rom` are actually used to create Roman numerals, so `\roman{cm}` might literally be just 1100 written backwards.
* Evidently `\rm` is deprecated and shouldn't be used, it has been replaced by `\textrm` -- I recently had to update a paper from using `\rm` to use `\textrm` because Pandoc wouldn't parse the deprecated markup.
* IIUC `\mathrm` can be used for making mathematical operators like `\sin`, `\cos`, etc, but I think that `\DeclareMathOperator{\ord}{ord}` is the preference?
* FYI: for variables that have more than one letter you can use [`\mathit{}`](https://tex.stackexchange.com/questions/218653/making-math-behaving-like-mathit) which _mostly_ fixes spacing? But [YMMV](https://en.wiktionary.org/wiki/your_mileage_may_vary) -- so in general my preference is to avoid these wordy variable names in typeset equations